385/Assets/Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Assets.Scripts;
+
+public class PlayerCamera : MonoBehaviour {
+    // player object camera is attached to
+    public GameObject player;
+
+    // offset of camera from player
+    private Vector3 offset;
+
+    // Static position of camera
+    private Vector3 initPos;
+
+    // camera that follows player
+    public Camera playerCam;
+
+    // checks Camera1 to change camera settings
+    public string CameraAxis = "Camera1";
```

Comment:
Add `"Camera1"` to the Util Axis file that contains all of the string constants.